### PR TITLE
fixes #17600 - Storybook does not draw chart

### PR DESCRIPTION
**Pull request by gailsteiger**

### CODE REVIEW

File: webpack/stories/index.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ import StatisticsChartsList from
 '../assets/javascripts/react_app/components/charts/StatisticsChartsList';
 import StatisticsChartBox from
 '../assets/javascripts/react_app/components/charts/StatisticsChartBox';
+import withDataProps from './data/statistics/WithDataProps';
```
Comment (ohadlevy) [inline webpack/stories/index.js]: should we call that with a different name (e.g. less generic than withDataProps?)

Comment (gailsteiger) [reply webpack/stories/index.js]: how about 
`
import data from './data/charts/donutChartMockData';
`

Comment (ohadlevy) [reply webpack/stories/index.js]: Shouldn't it be statistics Data vs just plain data?

**Comment by dLobatog** [comment]
Set to WoC as it's waiting on a question posed by @ohadlevy

**Comment by gailsteiger** [comment]
@ohadlevy I made the changes we discussed

File: .storybook/head.html [hunk 2]
```diff
@@ -5,7 +5,7 @@
 <script src='https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/js/bootstrap.min.js'></script>
 <script src='https://cdnjs.cloudflare.com/ajax/libs/d3/4.3.0/d3.min.js'></script>
 <script src='https://cdnjs.cloudflare.com/ajax/libs/c3/0.4.11/c3.min.js'></script>
-<script src='https://cdnjs.cloudflare.com/ajax/libs/patternfly/3.10.0/js/patternfly.min.js'></script>
+<script src='https://cdnjs.cloudflare.com/ajax/libs/patternfly/3.15.0/js/patternfly.min.js'></script>
```
Comment (ohadlevy) [inline .storybook/head.html:8]: it seems you need to set the minimum version of patternfly gem as well.

File: webpack/stories/index.js [hunk 3]
```diff
@@ -18,35 +17,37 @@ addDecorator((story) => (
...
       errorText="Ooops"
       status="ERROR" />
   ))
-  .add('With data', () => (
-    <StatisticsChartBox
-      config={{data: {columns: [1, 2]}}}
-      modalConfig={{}}
-      noDataMsg={'No data here'}
-      id="target"
+  .add('Donut Chart', () => (
```
Comment (ohadlevy) [inline webpack/stories/index.js:45]: can you also add a story for the modal chart?

**Comment by ohadlevy** [comment]
test foreman

**Comment by ohadlevy** [comment]
test katello

File: bundler.d/assets.rb [hunk 4]
```diff
@@ -1,7 +1,7 @@
 gem 'jquery-turbolinks', '~> 2.1'
 gem 'jquery-ui-rails', '< 5.0.0'
 gem 'autoprefixer-rails', '~> 5.2'
-gem 'patternfly-sass', '~> 3.9'
+gem 'patternfly-sass', '~> 3.15.0'
```
Comment (ohadlevy) [inline bundler.d/assets.rb:3]: @dLobatog am I correct to assume ~> 3.15 will work just as well?

**Comment by ohadlevy** [comment]
thanks @gailsteiger

**Pull request merged**